src/qt/automintnotification.h

```diff
@@ -36,4 +37,30 @@ private Q_SLOTS:
     LelantusModel *lelantusModel;
 };
 
+class AutomintSparkNotification : public QDialog
```

Comment:
Why we need separate type here? We can just check if Spark already active set spark model instead of lelantus model,